sample-prep/Biomek_pooling_VI.py

```diff
@@ -29,107 +43,287 @@ def main(process_id, file_id):
 
     lims.get_batch(inputs + outputs)
 
```

Comment:
from 30 to 44 could be a function, and could be put in a module, I think :)

Reply:
Sure could. I am sometimes storing slightly different things though. Sometimes I want both input and output in together as tuples in a list. Sometimes I also append some udf information directly in this loop rather than later. And the 'output-type' and 'output-generation-type' could be different from script to script. But probably could work that in.